README.md

```diff
@@ -216,6 +216,8 @@ The following is a list of static resources.
 - [azure_web_app_functions](docs/resources/azure_web_app_functions.md)
 - [azure_webapp](docs/resources/azure_webapp.md)
 - [azure_webapps](docs/resources/azure_webapps.md)
+- [azure_dns_zones_resource](docs/resources/azure_dns_zones_resource.md)
```

Comment:
you can run `bundle exec rake docs:resource_links` to position this resource in alphabetical order

Reply:
Wont it will create merge conflict ?

Reply:
Done.